hash/thinkpad8xx.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0-1.0
+-->
+<softwarelist name="thinkpad8xx" description="IBM Thinkpad 800 series software">
+
```

Comment:
I believe the official name for the line was “ThinkPad Power Series”.

Reply:
Yes! Fixed, thank!